packages/helm-charts/blockscout/templates/grafana-hook.yaml

```
@@ -0,0 +1,46 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ .Release.Name }}"
+  annotations:
+    "helm.sh/hook": post-upgrade
+spec:
+  template:
+    spec:
+      serviceAccountName: {{ .Release.Name }}-rbac
+      initContainers:
+{{ include "celo.blockscout.initContainer.secrets-init" . | indent 10 }}
```

Indentation is off, it should be on the same level as `initContainer` which is 6 spaces.
When the template is processed, it would look like:
```
initContainers:
- name: secrets-init
```
```suggestion
{{ include "celo.blockscout.initContainer.secrets-init" . | indent 6 }}
```